README.md

```diff
@@ -1 +1,186 @@
-# nexus-repository-apk
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2019-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+# Nexus Repository apk Format
+
+[![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-apk.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-apk%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-apk.svg?style=svg)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-apk) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apk/depshield.svg)](https://depshield.github.io)
+
+# Table Of Contents
+
+* [Developing](#developing)
+   * [Requirements](#requirements)
+   * [Building](#building)
+* [Using APK with Nexus Repository Manager 3](#using-apk-with-nexus-repository-manager-3)
+* [Compatibility with Nexus Repository Manager 3 Versions](#compatibility-with-nexus-repository-manager-3-versions)
+* [Installing the plugin](#installing-the-plugin)
+   * [Easiest Install](#easiest-install)
+   * [Temporary Install](#temporary-install)
+   * [(more) Permanent Install](#more-permanent-install)
+   * [(most) Permanent Install](#most-permanent-install)
+* [The Fine Print](#the-fine-print)
+* [Getting Help](#getting-help)
+
+## Developing
+
+### Requirements
+
+* [Apache Maven 3.3.3+](https://maven.apache.org/install.html)
+* [Java 8+](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
```

Comment:
Java8. remove + sign

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/1/commits/09471e269484ac8a27a67f73695f1a82de8bc47c